src/redux/books/books.js

```diff
@@ -1,7 +1,14 @@
+import { v4 } from 'uuid';
+
 const ADD = 'bookstore/books/ADD';
 const REMOVE = 'bookstore/books/REMOVE';
 
-export default function reducer(state = [], action) {
+const defaultBooks = [
+  { id: v4(), author: 'George Martin', title: 'Game Of Thrones' },
+  { id: v4(), author: 'Mario Mendoza', title: 'Satanas' },
+];
+
+export default function reducer(state = defaultBooks, action) {
   switch (action.type) {
     case ADD:
```

Comment:
- [ ] Unable to add or remove book details. As per your concern raised, I'm thinking the issue you are facing with errors emerged at an earlier stage.  I also noticed that the Eslint command cannot be successfully run on your app. I, therefore, suggest that you create a new project following a step-by-step guide to installing react-app, and carefully ensure that linters are working correctly, especially Eslint. Add the components/ modules one after the other, building the project bottom-up, ensuring each integration is working as intended. I believe with these steps the source of the error can easily be traced.


- [ ] OPTIONAL
It will be a nice idea to add a label to the _add new book button_ so as to improve the UX of your app.
![image](https://user-images.githubusercontent.com/12745474/162044241-83e21d3a-ddd6-42a0-92b6-e718cbd1348f.png)